### PR TITLE
Build update

### PR DESCRIPTION
**Pull request by liealex**

### CODE REVIEW

File: doc/itof/windows_build_instructions.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ cmake --build . --target install --config Release
 ```console
 git clone --branch v5.0.0 https://github.com/analogdevicesinc/ToF
 cd ToF
+git submodule ini
```
Comment (dNechita) [inline doc/itof/windows_build_instructions.md:54]: There is no 'ini' command. It should be 'init'.

File: doc/itof/nxp_build_instructions.md [hunk 2]
```diff
@@ -99,6 +101,8 @@ Please note, ensure you are using the intended branch.
 ```console
 git clone --branch v5.0.0  https://github.com/analogdevicesinc/ToF
 cd ToF
+git submodule ini
+git submodule update
```
Comment (dNechita) [inline doc/itof/nxp_build_instructions.md:105]: You can reduce to one line:
git submodule update --init

File: doc/itof/windows_build_instructions.md (hunk 1 above)
Comment (liealex) [reply doc/itof/windows_build_instructions.md]: Typo corrected

File: doc/itof/nxp_build_instructions.md (hunk 2 above)
Comment (liealex) [reply doc/itof/nxp_build_instructions.md]: Changed

File: README.md [hunk 3]
```diff
@@ -30,24 +30,14 @@ Other
...
 ## Supported Embedded Platforms
 
-For more details on building the SDK on an embedded platform please check the **User Guide** specified below:
-### ITOF camera
-| Operating system | Evaluation boards | Documentation | GitHub main status |
-| --------- | ----------- | ----------- | ----------- |
-| NXP | [EVAL-ADTF3175D-NXZ](https://wiki.analog.com/resources/eval/user-guides/eval-adtf3175d-nxz) | [Build instructions](doc/itof/nxp_build_instructions.md) | [![Build status](https://dev.azure.com/AnalogDevices/3DToF-rework/_apis/build/status/analogdevicesinc.ToF?branchName=main)](https://dev.azure.com/AnalogDevices/3DToF-rework/_build?view=runs&branchFilter=3310) |
-
-[How to write the SD card image onto the SD card?](doc/sdcard_burn.md)
+For more details on building the SDK on an embedded platform please check this [link.](https://github.com/analogdevicesinc/libaditof?tab=readme-ov-file#supported-embedded-platforms)
```
Comment (dNechita) [inline README.md:33]: Can you check if you could use relative paths instead of full paths?

Comment (dNechita) [reply README.md:33]: Actually, ignore this. I thought the PR was on libaditof repo, not on ToF.

**Review by dNechita — APPROVED** [review]

**Pull request merged**